uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/settings.js

```diff
@@ -15,6 +15,16 @@ define([
 
     }]);
 
+    services.factory('AppSettings',['$resource', 'config', '$log', function ($resource, config, $log) {
```

Comment:
A note for future: please run semistandard BEFORE changing the file, then apply changes, not vice versa.